front/src/components/boxs/device-in-room/device-features/NumberDeviceFeature.jsx

```diff
@@ -0,0 +1,50 @@
+import get from 'get-value';
+
+import { getDeviceName } from './utils';
+import { DeviceFeatureCategoriesIcon } from '../../../../utils/consts';
+
+const MultiLevelDeviceType = ({ children, ...props }) => {
```

Comment:
I guess it should be NumberDeviceFeature here :)

Reply:
No, I didn't copy/paste :p